_posts/en/newsletters/2021-08-04-newsletter.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Bitcoin Optech Newsletter #160'
+permalink: /en/newsletters/2021/08/04/
+name: 2021-08-04-newsletter
+slug: 2021-08-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes our regular sections describing how you
+can prepare for taproot, summarizing the latest releases and release
+candidates, and listing notable changes to popular Bitcoin
+infrastructure projects.
+
+## News
+
+*No significant news this week.*
+
+## Preparing for taproot #7: multisignatures
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/06-multisignatures.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.1rc1][C-Lightning 0.10.1] is a release candidate for
+  an upgrade that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  support for [bech32m address encoding][topic bech32], and enhanced support for
```

Comment:
Good spot. This wording is also in the 22.0 release notes (https://github.com/bitcoin-core/bitcoin-devwiki/wiki/22.0-Release-Notes-draft#updated-rpcs). I guess because it was backported from master to the 0.21.1 branch.